flink/v1.14/flink/src/test/java/org/apache/iceberg/flink/TestFlinkTableSink.java

```diff
@@ -96,14 +96,11 @@ public TestFlinkTableSink(String catalogName, Namespace baseNamespace, FileForma
   protected TableEnvironment getTableEnv() {
     if (tEnv == null) {
       synchronized (this) {
-        EnvironmentSettings.Builder settingsBuilder = EnvironmentSettings
-            .newInstance()
-            .useBlinkPlanner();
+        EnvironmentSettings.Builder settingsBuilder = EnvironmentSettings.newInstance();
         if (isStreamingJob) {
           settingsBuilder.inStreamingMode();
           StreamExecutionEnvironment env = StreamExecutionEnvironment
               .getExecutionEnvironment(MiniClusterResource.DISABLE_CLASSLOADER_CHECK_CONFIG);
-          env.setStreamTimeCharacteristic(TimeCharacteristic.EventTime);
```

Comment:
Is this also deprecated?

Reply:
Oh yeah that's also deprecated. That's the default value now.